src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableEntityContext.cs

```diff
@@ -44,6 +40,11 @@ public interface IDurableEntityContext
         /// </summary>
         bool IsNewlyConstructed { get; }
 
+        /// <summary>
+        /// Contains function invocation context to assist with dependency injection at Entity construction time.
+        /// </summary>
```

Comment:
I think there is a chance users will use this field for their own purposes once they discover it. That's probably fine. However, perhaps the summary should mention that this field is null if not running on NETSTANDARD2_0?